proto/poktroll/proof/params.proto

```diff
@@ -25,7 +25,10 @@ message Params {
   // being determined probabilistically via ProofRequestProbability.
   uint64 proof_requirement_threshold = 3 [(gogoproto.jsontag) = "proof_requirement_threshold"];
 
-  // TODO_UPNEXT(@bryanchriswhite, #579): Add proof_missing_penalty
+  // proof_missing_penalty is the number of tokens which should be slashed from a supplier
+  // when a proof is required (either via proof_requirement_threshold or proof_missing_penalty)
+  // but is not provided.
+  uint64 proof_missing_penalty = 4 [(gogoproto.jsontag) = "proof_missing_penalty"];
```

Comment:
Thought on making this type `cosmos.base.v1beta1.Coin`

Reply:
Perfect. :ok_hand: Thank you! :hugs: